src/modules/Library/features/Applet/Applet.utils.ts

```diff
@@ -34,6 +38,16 @@ export const getActivities = (
       isActivitiesExpanded = true;
     }
 
+    const isPerformanceTask = checkIfPerformanceTask(activity.items);
+    if (isPerformanceTask) {
```

Comment:
maybe !isPerformanceTask, or I'm wrong

Reply:
no

Reply:
because if ! isPerformanceTask we need to check items (what is done below)